eth/main.cpp

```diff
@@ -1137,14 +1201,14 @@ int main(int argc, char** argv)
 				return true;
 
 			string r = getResponse(_t.userReadable(isProxy,
-				[&](TransactionSkeleton const& _t) -> pair<bool, string>
-				{
-					h256 contractCodeHash = web3.ethereum()->postState().codeHash(_t.to);
-					if (contractCodeHash == EmptySHA3)
-						return std::make_pair(false, std::string());
-					// TODO: actually figure out the natspec. we'll need the natspec database here though.
-					return std::make_pair(true, std::string());
-				}, [&](Address const& _a) { return _a.hex(); }
+			                                       [&](TransactionSkeleton const& _t) -> pair<bool, string>
```

Comment:
Don't change the indentation like this, it's too much to the right, this is not always convenient to read